src/WebSite/Bit.Platform.WebSite/Web/Styles/abstracts/_mixins.scss

```diff
@@ -2,47 +2,7 @@
 @import '_media-queries.scss';
 @import '_colors.scss';
 
-@mixin SecondaryBtn {
-    display: flex;
-    justify-content: center;
-    align-items: center;
-    width: rem(213px);
-    height: rem(56px);
-    border-radius: rem(4px);
-    background-color: $White;
-    border: rem(2px) solid $B2Color;
-    box-sizing: border-box;
-    font-weight: 600;
-    font-size: rem(18px);
-    line-height: rem(32px);
-    color: $B2Color;
-    cursor: pointer;
-
-    &:hover {
-        background-color: $Gray5;
-        border-color: $B3Color;
-        color: $B3Color;
-    }
-
-    @include lg {
-        width: rem(185px);
-        height: rem(48px);
-        font-size: rem(16px);
-        line-height: rem(24px);
-    }
-
-    @include lt-lg {
-        width: rem(146px);
-        height: rem(40px);
-        font-size: rem(12px);
-        line-height: rem(16px);
-    }
-
-    @include sm {
-        width: rem(118px);
-    }
-}
-
+/*---------------defualt sizes----------------*/
 @mixin subtitle1 {
```

Comment:
All the mixins should have a CapitalCase name

Reply:
The same is valid for "@mixin lg"  and "@mixin sm" ,...?

Reply:
nope, they're defined based on a best practice for media query mixins.